README.md

```diff
@@ -700,8 +700,7 @@ profiles:
 ### RemoveFailedPods
 
 This strategy evicts pods that are in failed status phase.
-You can provide an optional parameter to filter by failed `reasons`.
-`reasons` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.
+You can provide optional parameters to filter by failed pods' `reasons`. and/or `exitCodes`. `exitCodes` apply to pods with `terminated` state only. `reasons` and `exitCodes` can be expanded to include reasons of InitContainers as well by setting the optional parameter `includingInitContainers` to `true`.
```

Comment:
is it really `and/or`?

if reasons doesn't match, then it returns an err:

https://github.com/kubernetes-sigs/descheduler/blob/b95380641fcdb31dee6026d7e9e96f226dbbd424/pkg/framework/plugins/removefailedpods/failedpods.go#L146-L147